### ISSUE TITLE
What sample size should be used for meta-analyses?

### ISSUE THREAD

**Issue by jdblischak** [ghm17/LOGODetect#9]
LOGODetect requires a single sample size for each GWAS. Could you please advise me on which sample size I should use for a meta-analysis where the sample size differs per SNP?

I looked at your Supplementary Table 12, and from the titles alone it appears that at least 2 of the studies were meta-analyses.  [Nagel et al.](https://doi.org/10.1038/s41588-018-0151-7) report a total sample size of 449,484, but your table says the sample size you used was 390,278. [Savage et al.](https://doi.org/10.1038/s41588-018-0152-6) report a total sample size of 269,867, which matches the value in your table.

In other words, should I use the max sample size in the meta-analysis (as you did for the Savage et al. study) or is there a different procedure I should use to determine the sample size for use with LOGODetect? Thanks!

**Comment by ghm17** [comment]
The sample size of 449,484 reported in Nagel et al. comprises 59,206 individuals from 23andMe. The released GWAS summary stats only include 390,278 individuals not in 23andMe.  

Currently, LOGODetect assumes homogeneous sample size across all SNPs (this assumption could be relaxed, but certain modifications are required, and the software will be updated in the future). In practice, we recommend to use the max sample size in the meta-analysis, and filter out those SNPs whose sample size is relatively much smaller (e.g. 50% max sample size).
